src/GovUk.Education.ExploreEducationStatistics.Admin/Services/FileUploadsValidatorService.cs

```diff
@@ -195,13 +114,95 @@ private bool IsFileExisting(Guid releaseId, ReleaseFileTypes type, string name)
                 .Any(rf => String.Equals(rf.ReleaseFileReference.Filename, name, CurrentCultureIgnoreCase)
                 && rf.ReleaseFileReference.ReleaseFileType == type);
         }
+
+        private async Task<Either<ActionResult, bool>> ValidateDataFileNames(Guid releaseId, string dataFileName, string metaFileName)
+        {
+            if (string.Equals(dataFileName.ToLower(), metaFileName.ToLower(), OrdinalIgnoreCase))
+            {
+                return ValidationActionResult(DataAndMetadataFilesCannotHaveTheSameName);
+            }
+            
+            if (dataFileName.IndexOf(" ", Ordinal) > -1 || 
+                dataFileName.IndexOfAny(Path.GetInvalidFileNameChars()) > -1)
```

Comment:
This is just a duplicate of the `FileContainsSpacesOrSpecialChars` directly below.

Reply:
yep - done